fgpyo/sam/__init__.py

```diff
@@ -668,6 +668,35 @@ def build(
             return PairOrientation.RF
 
 
+def isize(r1: AlignedSegment, r2: Optional[AlignedSegment] = None) -> int:
+    """Computes the insert size for a pair of records."""
```

Comment:
Could you update docs to explain that the second arg is optional and when it will fail if the second record isn't provided?